source/documentation/other-topics/custom-domain-cert.md

```diff
@@ -7,28 +7,35 @@ cluster, this would be `*.apps.live-1.cloud-platform.service.justice.gov.uk`. As
 long as it is defined on the `Ingress` resource, it works automatically with a
 wildcard TLS certificate.
 
-However, most applications will typically need to be served on a `gov.uk`
-hostname. These hostnames (or usually, DNS zones) are managed externally,
-relative to the cluster, and there is a number of actions in order to set them
+However, most applications will typically need to be served on their own,
+application-specific `gov.uk` hostname. These hostnames (or usually, DNS zones)
+are managed individually and there is a number of actions in order to set them
 up for usage.
 
 #### Setup
 
-##### You do not have a DNS zone for the desired hostname
-Please [create a support ticket](http://goo.gl/msfGiS) providing as much
-information as possible.
+##### Defining the DNS zone
```

Comment:
It would be good to have a sentence explaining what the DNS zone is for, for people who don't fully understand how the pieces fit together.